.travis.yml

```diff
@@ -8,15 +8,8 @@ env:
 addons:
   postgresql: 9.6
 
-rvm:
-  - 2.5.0
-
 dist: trusty
 
-before_install:
-  - gem install bundler -v ">= 1.8.9"
-
-before_script:
-  - ./bin/rails db:drop db:setup
+before_script: bundle exec rails db:reset
 
-script: ./bin/rspec
+script: bundle exec rspec
```

Comment:
What's the reasoning for changing to `bundle exec` over the binstub? I though binstubs were the preferred way to run binaries because they abstract away the implementation (ie. Bundler).

Reply:
Reverted in a fixup.